db/sequence_allocator.go

```diff
@@ -109,7 +110,7 @@ func (s *sequenceAllocator) incrWithRetry(key string, numToReserve uint64) (uint
 	//       is nil by the time we log the warning above.  This seems most likely to be a race/scope issue with the callback processing
 	//       in the go-couchbase Incr/Do, and the sleep after the last attempt above.  Forcing the error to non-nil here to ensure we don't
 	//       proceed without an error in this case.
-	return 0, fmt.Errorf("Unable to increment sequence: %v", err)
+	return 0, pkgerrors.Wrapf(err, "Unable to increment sequence")
```

Comment:
I think this wrap is probably redundant (method name in the stack track will make it clear that this is a failed sequence incr).

Reply:
As it stands, there is no way to get a stack trace that I know of .. unless we pkgerrors wrap it here or lower.  (this was my motivation of using pkgerrors gobally in the codebase, so everything would get a stack trace by default)

I looked into the source of this error, and it's probably worth having a screenhero chat to walk through this code together.

Reply:
I pushed this one down to `bucket_gocb.Incr()`

Reply:
Pushed down to `bucket_gocb.Get()`